service/src/main/java/org/cbioportal/genome_nexus/service/VariantAnnotationService.java

```diff
@@ -47,7 +47,7 @@
     VariantAnnotation getAnnotation(String variant)
         throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException;
     List<VariantAnnotation> getAnnotations(List<String> variants);
-    VariantAnnotation getAnnotation(String variant, String isoformOverrideSource, Map<String, String> token, List<String> fields)
+    VariantAnnotation getAnnotation(String variant, String isoformOverrideSource, Map<String, String> token, List<AnnotationType> annotationTypeList)
```

Comment:
same as above, and for the rest of the PR where applicable. Let's keep the param as is for now?

```suggestion
    VariantAnnotation getAnnotation(String variant, String isoformOverrideSource, Map<String, String> token, List<AnnotationType> fields)
```